factory-hooks/post-settings-php/acsfd8+.memcache.settings.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @file
+ * Contains caching configuration.
+ */
+use Composer\Autoload\ClassLoader;
+
+/**
+ * Use memcache as cache backend.
+ *
+ * Autoload memcache classes and service container in case module is not
+ * installed. Avoids the need to patch core and allows for overriding the
+ * default backend when installing Drupal.
+ *
+ * @see https://www.drupal.org/node/2766509
+ */
+
+if (getenv('AH_SITE_ENVIRONMENT') &&
+  array_key_exists('memcache', $settings) &&
+  array_key_exists('servers', $settings['memcache']) &&
+  !empty($settings['memcache']['servers'])
+) {
+
+// Check for PHP Memcached libraries.
+$memcache_exists = class_exists('Memcache', FALSE);
```

Comment:
Should this code block be indented since it's wrapped in an if?

Reply:
Got it